kotlintest-assertions/src/main/kotlin/io/kotlintest/matchers/string/matchers.kt

```diff
@@ -154,10 +159,26 @@ infix fun String?.shouldNotHaveMinLength(length: Int) = this shouldNot haveMinLe
 fun haveMinLength(length: Int) = neverNullMatcher<String> { value ->
   Result(
       value.length >= length,
-      "${convertValueToString(value)} should have minimum length of $length",
-      "${convertValueToString(value)} should have maximum length of ${length - 1}")
+      "${value.show()} should have minimum length of $length",
+      "${value.show()} should have maximum length of ${length - 1}")
 }
 
+
+fun String?.shouldHaveLengthBetween(min: Int, max: Int) = this should haveLengthBetween(min, max)
```

Comment:
Perhaps we can have an overload that uses range? it would allow a infix notation

```
"foo" shouldHaveLengthIn 1..5
```

Reply:
Good idea. I've added that.